samples/ScratchIslandApp/WindowExe/WindowExe.def

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Why have the empty def file at all?

Reply:
I have no idea why we have that in WindowsTerminal.def, but I didn't want to tempt fate